### DISCUSSION TITLE
New Member - Do I need a VPN/Proxy to run this?

### DISCUSSION THREAD

**Discussion by binaryjson** [jef/streetmerchant#2381] [Help]
Hi,

I am one of the frustrated consumers trying to buy a GPU (RTX3060/70/80/90) for the last few months with no luck and just happened to come across streetmerchant. 

I was wondering if it's safe to run this program in my PC (to scan and add items to cart from Amazon or BestBuy or Walmart without being blocked by merchants). 

If not, how do others do this?

Thanks!

**Comment by Mr-Tech-13** [comment]
You could use a vpn
